app/controllers/concerns/versioning.rb

```diff
@@ -9,10 +9,7 @@ def version_param
   end
 
   def version_number
-    @version_number = extract_version(version_param)
-  end
-
-  def full_api_version_number
-    "#{major_version_number(version_number)}.#{minor_version_number(version_number)}"
+    extracted_from_param = extract_version(version_param)
+    @version_number = "#{major_version_number(extracted_from_param)}.#{minor_version_number(extracted_from_param)}"
```

Comment:
```suggestion
    @version_number = full_version_number_from(extracted_from_param)
```
Use the helper method from `VersioningHelpers`?

Reply:
I wondered whether `version_number` could/should be memoised. (I guess it wasn't before).